doc/tutorial_sector.rst

```diff
@@ -0,0 +1,637 @@
+.. SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _tutorial_sector:
+
+##########################################
+Tutorial: Sector-Coupled
+##########################################
+
+.. note::
+    If you have not yet installed PyPSA-Earth, please refer to the :ref:`installation` section.
+
+In this tutorial, we will show you how to run the sector-coupled model. The sector-coupled model
+is a model that considers the energy system as a whole, including the electricity, heat, transport,
+and industry sectors. The model is based on the PyPSA-Eur model, which is a model of the European
+energy system. The sector-coupled model is a global model that can be used to model any region
+of the Earth. This section explains how to run and analyze the tutorial model.
+
+
+The sector-coupling code can be run as an overnight / greenfield scenario or myopic scenario.
+The overnight scenario is a long-term scenario that runs for a year, while the myopic scenario
+is a short-term scenario that runs for a day.
+
+
+Overnight Scenarios
+=============================================
+
+Configuration
+---------------------------------------------
+
+The configuration for sector-coupled is present in the  ``config.default.yaml`` file.
+This foresight parameter is set to overnight by default.
+
+.. code:: yaml
+
+    foresight: overnight
+
+Documentation for all options will be added successively to :ref:`config`.
+
+Scenarios can be defined like for electricity-only studies, but with additional wildcard options.
+
+.. code:: yaml
+
+    scenario:
+    simpl: [""]
+    ll: ["copt"]
+    clusters: [10]
+    opts: [Co2L-3h]
+    planning_horizons: # investment years for myopic and perfect; or costs year for overnight
+    - 2030
+    sopts:
+    - "144h"
+    demand:
+    - "AB"
+
+For allowed wildcard values, refer to :ref:`wildcards`.
+
+Execution
+---------------------------------------------
+To run the tutorial for the sector-coupled model, you need to activate the pypsa-earth environment.
+You need to have installed PyPSA-Earth using the instructions provided in the :ref:`installation` section.
+Make sure to be in the PyPSA-Earth root directory and run the following command:
+
+.. note::
+
+    It is good practice to perform a dry-run using the option -n, before you commit to a run:
+
+    .. code:: bash
+
+        .../pypsa-earth (pypsa-earth) $ snakemake solve_sector_networks -j8 --configfile config.tutorial.yaml -n
```

Comment:
Just to be sure, is `-j8` failure-safe? Will it work properly also for the machines with a lower number of cores?

Reply:
I think it best to reduce it to cores of 2 as it is in other part of the documentation

Reply:
Great, thank you!